### PR TITLE
Update readme with more information.

### PR DESCRIPTION
**Pull request by nadiaya**
Update readme with more information.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+
+All Dockerfiles use deep learning framework images provided by SageMaker as their "base" images.
+
+These "base" images are specified with the following naming convention:
+
+::
+
+    # 520713654638.dkr.ecr.<region>.amazonaws.com/sagemaker-<framework>:<framework_version>-<processor>-py3
+    520713654638.dkr.ecr.<region>.amazonaws.com/sagemaker-<tensorflow-scriptmode_or_mxnet>:<framework_version>-<cpu_or_gpu>-py3
```
Comment (laurenyu) [inline README.rst]: I'd "uncomment" the first line, and then turn the second line into an explanation that `framework` can be either `tensorflow-scriptmode` or `mxnet` and then the same for `processor`. The lone sentence below will then also flow a little better

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,260 @@
+==============================
+Amazon SageMaker RL Containers
+==============================
+
+A set of Dockerfiles that provides Reinforcement Learning (RL) solutions to be used in SageMaker.
```
[the remark this answers is not in the document]
Comment (nadiaya) [reply README.rst]: libraries? 
environments can be mistaken with 'simulation environments' which we do not provide and expect user to bring themselves.

[the remark this answers is not in the document]
Comment (laurenyu) [reply README.rst]: how about replacing "provides" with "enables"?

[the remark this answers is not in the document]
Comment (nadiaya) [reply README.rst]: nice!

**Pull request merged**